brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/VIP/Coordinator.swift

```diff
@@ -16,12 +16,12 @@ protocol BaseControllable: UIViewController {
 protocol Coordinatable: CoordinatableRoutes {
 
     var childCoordinators: [Coordinatable] { get set }
-    var navigationController: UINavigationController { get set }
+    var navigationController: RootNavigationController { get set }
```

Comment:
why? a normal NVC reference can still hold a subclass off it...
does the "rootNVC" give us that much that we want to lock the variable type to it?